cfgov/unprocessed/apps/teachers-digital-platform/css/pages/survey.less

```diff
@@ -161,6 +161,9 @@ ul.tdp-survey__choice-question {
 
 .tdp-survey-sidebar {
     &.o-sidebar-content {
+        .respond-to-max(@bp-sm-max, {
+            border-bottom: 1px solid #b4b5b6;
```

Comment:
Can we use color variables here? `border-bottom: 1px solid @gray-40`

https://cfpb.github.io/design-system/foundation/color